app/assets/stylesheets/config/_generator.scss

```diff
@@ -60,79 +60,18 @@
     'border-radius',
     ('default': var(--radius), '0': 0, 'full': 9999px),
     (),
-    true
```

Comment:
Are all these key/value pairs being removed because we literally never use these classes at the moment in the codebase, i.e. manual CSS purge?

Reply:
Correct. And I removed them because:
- in case we end up using Tailwind library we gonna use PurgeCSS anyway so those would be gone 🤷 
- in case we don't end up using Tailwind library, these classes weren't used anyway. If we ever need one, we can always manually add.